src/models/StandaloneSimilarity.py

```diff
@@ -0,0 +1,21 @@
+import random
+
+from cleaning_functions import pandas_vector_to_list
```

Comment:
that's fair enough actually, we can just patch the function to do something more sensible